src/components/workflow/SelectWorkflow.vue

```diff
@@ -130,13 +142,22 @@ export default {
       console.log("Submitting workflow");
       let self = this;
       self.workflowSubmission.loading = true;
+      self.errors = [];
       await self.workflowService.submitWorkflow(this.workflowSubmission.selectedWorkflow, self.workflowService.getSelectedPrimaryfileIds(this.selectedFiles))
          .then(response => {
             let jobsobj = response.data;
-            this.jobs = new Map(Object.entries(jobsobj));
+
+            console.log(jobsobj);
```

Comment:
Did you mean to leave this console.log in? No worries if so, just wondering if it isn't a leftover from testing.